.github/workflows/python-package.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         # If you wish to install the version of fparser pointed to by the
         # submodule instead of the released version (from pypi) then
         # uncomment the following line:
-        pip install external/fparser
+        # pip install external/fparser
```

Comment:
I assume this is commented as it points to a wrong path?

Reply:
Nope - this line is needed only when the fparser submodule included with PSyclone needs to be installed. When we're working with a released version of fparser (as we must do when creating a release of PSyclone) then it will be installed automatically as part of the pip installation of PSyclone itself.

Reply:
Thanks for explaining!